### ISSUE TITLE
Update team member bios on website to replace 1-2-1 new starter meetings

### ISSUE THREAD

**Issue by heyhaleema** [GSTT-CSC/csc-test-site#244]
The number of 1-2-1 meetings new starters need to arrange and allocate time to has significantly increased due to the team's expansion.

I suggest (1) team members update their bios on the website to include enough detail to make these meetings mostly redundant and (2) new starters are required to meet with 3 - 5 members as part of their onboarding depending e.g., based on reading each of the team member's bio, they reach out to those involved with areas of work they're interested/would like to further develop competencies in.

- [ ] @hshuaib90 
- [ ] @dangerdika 
- [x] @simoambr 
- [x] @laurencejackson 
- [ ] @AnilMistry 
- [x] @alexandertdeng 
- [ ] @tomaroberts 
- [x] #346
- [ ] @kavithavaran 
- [ ] @aysha1207 
- [x] @Beatrix-Fletcher 
- [ ] @AgatheZ 
- [ ] @sophie22 
- [ ] @robd78 
- [ ] @mikewoodward94

**Comment by Beatrix-Fletcher** [comment]
Agreed, I will work on mine - thoughts on having a rough template of 'things to cover in your bio' or 'purpose of bio' to really be clear on the sort of information to include and in what context? 

Additionally was thinking that we should all include clinical backgrounds (where applicable)- for sure in the long form bio, but possibly also in the short from (name, role, clinical)? Thought isn't fully fleshed out re wording...for example: Clinical Registration: X (Past) or Y (Current)?

**Comment by hshuaib90** [comment]
Alex has updated his with more detail. Do people think this is a good format to follow? @heyhaleema @Beatrix-Fletcher ?

**Comment by dangerdika** [comment]
No longer relevant
